darwin/path_utils.py

```diff
@@ -41,3 +44,49 @@ def deconstruct_full_path(filename: str) -> Tuple[str, str]:
     """
     posix_path = PurePosixPath("/") / filename
     return str(posix_path.parent), posix_path.name
+
+
+def parse_manifest(path: Path) -> dict:
+    """
+    Parses the given manifest and returns a list of all the properties in it.
+
+    Parameters
+    ----------
+    path : str
+        The path to the manifest.
+
+    Returns
+    -------
+    list[Property]
+        A list of all the properties in the given manifest.
+    """
+    with open(path) as f:
+        manifest = json.load(f)
+
+    return manifest
+
+
+def is_properties_enabled(
```

Comment:
While calling this function - would we have `annotations` (list of file-paths) available as the function argument? If not, I could re-use the function [_get_files_for_parsing](https://github.com/v7labs/darwin-py/blob/master/darwin/importer/importer.py#L168-L173) to get the annotation files?

Reply:
yes, the intention is this will be called at the top level path, which includes the .v7 hidden folder + the metadata.json file. The annotations will be a sub-folder of that parent
-> path_to_send_to_function
 |-> .v7
 |--> metadata.json
 |-> annotations
 |--> *.json